src/IceNucleation.jl

```diff
@@ -108,6 +109,31 @@ function ABIFM_J(
     return max(FT(0), FT(10)^logJ * FT(1e4)) # converts cm^-2 s^-1 to m^-2 s^-1
 end
 
+"""
+    INP_concentration_frequency(INPC,T)
+
+ - `params` - a struct with INPC(T) distribution parameters
+ - `INPC` - concentration of ice nucleating particles [m^-3]
+ - `T` - air temperature [K]
+
+Returns the relative frequency of a given INP concentration,
+depending on the temperature.
+Based on Frostenberg et al., 2023. See DOI: 10.5194/acp-23-10883-2023
+"""
+function INP_concentration_frequency(
+    params::CMP.Frostenberg2023,
+    INPC::FT,
+    T::FT,
+) where {FT}
+
+    T_celsius = T - 273.15
+    (; σ, a, b) = params
```

Comment:
Hi, this is Hannah Frostenberg, the author of the original manuscript :)
Just a small comment from my side: the parameters a and b in eq. 1 of the paper were essentially only to get the units right. They were not intended to have any other numerical values than 1. So maybe it would be good to remove them from the list of parameters (if I read Julia correctly, I have not used that language so far).

Reply:
Hi! It's great to meet you on GitHub. I'm Anna Jaruga and I'm working on different cloud microphysics parameterizations at CliMA. I'm looking forward to trying your parameterization. - We don't really have an aerosol model, so this observation based approach is a really interesting opportunity to try.

We should soon have a 0-dimensional adiabatic parcel example to test the implementation and look at the `dt` and drawing frequency sensitivity. Could I message you when we have some plots to show?

And thank you for the parameters suggestion. Yes, I agree, we should not be passing them in. (I wish we could write all parameterizations non-dimensional. That would make model implementation so much easier.) Thank you for being clear on what the units are. Sometimes it's really hard to implement parameterizations based on papers, because the units and constants are not clear.

Reply:
Hi Anna,
Thank you for your message! It will be superinteresting to see the results once you have the implementation in the parcel example! Please do message me and Luisa.